### ISSUE TITLE
Insufficient contrast of focus indicator in menus

### ISSUE THREAD

**Issue by craxal** [electron/electron#45068]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

33.3.0

### What operating system(s) are you using?

Windows

### Operating System Version

Windows 11

### What arch are you using?

x64

### Expected Behavior

A border with appropriate contrast should be shown as a focus indicator when using the keyboard to navigate menus. This is how apps like File Explorer and Edge work:

![Image](https://github.com/user-attachments/assets/a0d493c8-06be-436b-9ae6-f4d65ef37df6)

Notice the highlight here is a hover indicator which does not need to have sufficient contrast.

### Actual Behavior

Only a highlight background in insufficient contrast with the surrounding colors is shown as a focus indicator. This is a violation of accessibility guidelines.

![Image](https://github.com/user-attachments/assets/a3a15037-676e-4a95-bc66-a513aae276c2)

### Additional Information

This is easily reproduced using the default project upon opening Electron Fiddler.

**Comment by nikwen** [comment]
#45033 suggests menu styles are inherited from Chromium upstream.

I don't think Electron changes anything about these menus, so the bug would probably need to be filed in the Chromium bug tracker.

**Comment by VerteDinde** [comment]
@craxal To @nikwen's point, could you confirm whether or not the behavior reproduces in Chrome? If yes, we should file a report upstream for accessibility. If no, we can take a look on the Electron side

**Comment by craxal** [comment]
@VerteDinde Well, I checked against Edge, which is Chromium-based, but Edge does not appear to have that problem.

**Comment by craxal** [comment]
Just to be clear, the highlight background color is identical but totally fine, because it doesn't need to contrast. It's just the focus (visible when using keyboard) that needs work.

**Comment by nikwen** [comment]
@craxal Thanks for checking!

Given that Chrome has the same issue and Electron just uses Chromium's code, would you be able to file this accessibility issue in the Chromium bug tracker? When Chromium fixes it, Electron will pick up the change as well.

(Edge probably has its own logic for menus. That's why it works differently there.)

**Comment by craxal** [comment]
Looks like there's already an issue open: https://issues.chromium.org/issues/40693702

**Comment by nikwen** [comment]
Are there any known strategies to get Chromium to prioritize a stale issue?

Commenting could be a good idea to let them know the issue still exists.

**Comment by felixrieseberg** [comment]
I'll close this issue for now as an upstream problem, but I'd like to point out that you _could_ write native code to build menus just like the one in Edge. 

We could fix this with Electron taking over custom menus, but that too is probably best implemented with a userland module. In absence of that, I think we'd also welcome an RFC by someone eager to build a net-new menu system!
